bin/test_travis.sh

```diff
@@ -73,7 +74,11 @@ EOF
     exit 0
 fi
 
-CXXFLAGS="-Werror" cmake $cmake_line ${SOURCE_DIR}
+if [[ "${TRAVIS_OS_NAME}" != "osx" ]] && [[ "${CC}" == *"clang"* ]]; then
+    CXXFLAGS="CC=clang-3.4 CXX=clang++-3.4" "-Werror" cmake "-stdlib=libc++" $cmake_line ${SOURCE_DIR}
```

Comment:
This line is completely messed up (many unrelated syntax errors)... Try this:
```
CC=clang-3.4 CXX=clang++-3.4 CXXFLAGS="-Werror -stdlib=libc++" cmake $cmake_line ${SOURCE_DIR}
```
That should fix the problem in https://travis-ci.org/sympy/symengine/jobs/68933084:
```
+CXXFLAGS='CC=clang-3.4 CXX=clang++-3.4'

+-Werror cmake -stdlib=libc++ -DCMAKE_INSTALL_PREFIX=/home/travis/our_usr -DCOMMON_DIR=/home/travis/our_usr -DCMAKE_BUILD_TYPE=Debug -DWITH_BFD=yes -DWITH_PYTHON=yes -DWITH_SYMENGINE_ASSERT=yes /home/travis/build/sympy/symengine

bin/test_travis.sh: line 78: -Werror: command not found
```